File: src/main/java/com/sendsms/africastalking/model/PojoSms.java
package com.sendsms.africastalking.model;

import javax.validation.constraints.NotEmpty;

/**
 * @author Christian Amani on 05/04/2019.
 */
public class PojoSms {

    @NotEmpty
    private String number;

    @NotEmpty
    private String message;

    public String getNumber() {
        return number;
    }

    public void setNumber(String number) {
        this.number = number;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}

File: webui/src/models/pojosms.js
class PojoSms {
    constructor() {
        this.message = '';
        this.number = '';
    }

    set content(message) {
        this.message = message;
    }

    get content() {
        return this._message;
    }

    get numbers() {
        return this.number;
    }

    set numbers(value) {
        this.number = value;
    }
}


export {PojoSms};